src-backbone/app/js/views/builder/flowchartView.js

```diff
@@ -0,0 +1,129 @@
+const FlowchartJS = require('flowchart.js');
+
+module.exports = Marionette.LayoutView.extend({
+
+    template: require('templates/builder/flowchartView'),
+
+    onAttach: function() {
+        const getFirstCondition = node => {
+            if (!node) {
+                return '';
+            }
+
+            let condition;
+            if (!(node.isCriteriaNode() && node.get('criteria_element') > 0)) {
+                condition = '';
+            } else {
+                try {
+                    condition = node.getElementQuestion();
+                } catch (ignore) {
+                    condition = '';
+                }
+            }
+
+            if (condition !== '') {
+                return condition;
+            }
+
+            for (let child of node.childrenNodes.models) {
+                condition = getFirstCondition(child);
+                if (condition !== '') {
+                    return condition;
+                }
+            }
+            return '';
+        };
+
+        const splitStringIntoLines = (str, l) =>{
```

Comment:
it seems like this can be replaced by str.split(' ').join('\n');

Reply:
I should document this. This function splits a string into lines up to `l` characters long, so the string won't just be one really long line.